proto-google-cloud-bigquerystorage-v1beta1/clirr-ignored-differences.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- see http://www.mojohaus.org/clirr-maven-plugin/examples/ignored-differences.html -->
+<differences>
+  <difference>
+    <differenceType>4001</differenceType>
+    <className>com/google/cloud/bigquery/storage/v1beta1/**</className>
```

Comment:
This could also be more specific. I just wrote it this broadly to get something to work. Turns out the problems wasn't what I first thought it was though.

Reply:
will be narrowing down to Storage*